src/Api/Controller/Changes.php

```diff
@@ -90,34 +92,45 @@ public static function publish(ModelWithContent $model, array $input): array
 	 */
 	public static function save(ModelWithContent $model, array $input): array
 	{
-		// we need to run the input through the form
-		// class to get a set of storable field values
-		// that we can send to the content storage handler
-		$form = Form::for($model, [
-			'ignoreDisabled' => true,
-			'input'          => $input,
-		]);
+		// Removes the old .lock file when it is no longer needed
+		// @todo Remove in 6.0.0
+		static::cleanup($model);
 
+		// get the current language
+		$language = Language::ensure('current');
+
+		// create the fields instance for the model
+		$fields = Fields::for($model, $language);
+
+		// get the changes and latest version for the model
 		$changes = $model->version(VersionId::changes());
 		$latest  = $model->version(VersionId::latest());
 
-		// Removes the old .lock file when it is no longer needed
-		// @todo Remove in 6.0.0
-		static::cleanup($model);
+		// get the source version for the existing content
+		$source  = $changes->exists($language) === true ? $changes : $latest;
```

Comment:
As it doesn't look that we use `$latest` otherwise, would maybe be better to only initialise it when changes does not exist.

Reply:
We use it once more further down for the isIdentical check